desc/compute/_bootstrap.py

```diff
@@ -381,6 +381,38 @@ def _compute_J_dot_B_Redl(params, transforms, profiles, data, **kwargs):
 
     helicity = kwargs.get("helicity", (1, 0))
     helicity_N = helicity[1]
-    j_dot_B_data = j_dot_B_Redl(geom_data, profile_data, helicity_N)
-    data["<J*B> Redl"] = grid.expand(j_dot_B_data["<J*B>"])
+    J_dot_B_data = compute_J_dot_B_Redl(geom_data, profile_data, helicity_N)
+    data["<J*B> Redl"] = grid.expand(J_dot_B_data["<J*B>"])
+    return data
+
+
+@register_compute_fun(
+    name="current Redl",
+    label="\\frac{2\\pi}{\\mu_0} I_{Redl}",
+    units="A",
+    units_long="Amperes",
+    description="Net toroidal current enclosed by flux surfaces, "
+    + "consistent with bootstrap current from Redl formula",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="r",
+    data=["rho", "psi_r", "p_r", "current", "<|B|^2>", "<J*B> Redl"],
+)
+def _current_Redl(params, transforms, profiles, data, **kwargs):
+    """Compute the current profile consistent with the Redl bootstrap current.
+
+    Compute the current using Equation C3 in
+    Landreman and Catto, Physics of Plasmas 19, 056103 (2012).
+    This is the same approach as STELLOPT VBOOT with SFINCS, and should be used in an
+    iterative method to update the current profile until self-consistency is achieved.
+    """
+    rho = transforms["grid"].compress(data["rho"])
+    current_r = transforms["grid"].compress(
+        -mu_0 * data["current"] / data["<|B|^2>"] * data["p_r"]  # perpendicular
```

Comment:
could save a bit of work by compressing these before doing out all the math but not a big deal most likely